platform/darwin/src/MGLRasterSource.h

```diff
@@ -1,93 +1,84 @@
-#import "MGLSource.h"
-#import "MGLTypes.h"
+#import "MGLTileSource.h"
 
 #import <CoreGraphics/CoreGraphics.h>
 
-@class MGLTileSet;
-
 NS_ASSUME_NONNULL_BEGIN
 
 /**
+ An `NSNumber` object containing a floating-point number that specifies the
+ width and height (measured in points) of each tiled image in the raster source.
+ 
+ This number also corresponds to the width and height (measured in points) at
+ which the map displays each tile when the map’s zoom level is an integer. The
+ source scales its images up or down when the map’s zoom level falls between two
+ integers.
+ 
+ This option applies only to tiles hosted by Mapbox and accessed through version
+ 4 of the
+ <a href="https://www.mapbox.com/api-documentation/#maps">Mapbox Maps API</a>,
+ such as through `mapbox://<mapid>` URLs. This option has no effect on tiles
```

Comment:
On the contrary, this option is relevant and does have an effect for third-party tile servers. In fact, most sources which use a third-party tile service will want to specify a value of `256`; if a tile service provides 256 pixel tiles and the source uses a value of `512`, tiles will appear at twice their intended resolution and appear blurry or pixelated.